llvm/include/llvm/SYCLLowerIR/SYCLPropagateAspectsUsage.h

```diff
@@ -0,0 +1,28 @@
+//===---- PropagateAspectsUsage.h - PropagateAspectsUsage Pass ------------===//
```

Comment:
Please change name here as well.

Reply:
Fixed in 3b4d9db4727ce6c7b516946df8fbfa680fd65a25